src/vehicle_use.cpp

```diff
@@ -1967,35 +1948,38 @@ void vehicle::interact_with( const tripoint &pos, int interact_part )
 
     selectmenu.addentry( EXAMINE, true, 'e', _( "Examine vehicle" ) );
     selectmenu.addentry( TRACK, true, keybind( "TOGGLE_TRACKING" ), tracking_toggle_string() );
-    if( has_controls ) {
+    if( vp_controls ) {
```

Comment:
A lot of the variable renames feel like churn for the sake of churn.  They're better, I guess, but not better enough to justify doing at this time

Reply:
Variables are of different type now - has_ were bools while these are vehicle parts

I agree there's no point in renaming for sake of renaming but I'd like to appeal that there's a case for clarity on what the variable is and as I've mentioned these changes are a part of a larger change I'd like to see in:
https://github.com/CleverRaven/Cataclysm-DDA/pull/44700

A large chunk of the `interact_with` is changed there and those are part of the changes, if it's a problem I can try rolling them back but I'm not sure if it'll make the PRs cleaner

Reply:
I'm not sure it's that much better in context, but I'm willing to give a pass because you are changing the type and it is unifying the function overall.  Just think carefully about doing this kind of thing.